docs/windows.md

```diff
@@ -1,57 +1,64 @@
 # Deploying Antrea on Windows
 
 ## Overview
-Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay
-network to forward packets between Nodes and implements NetworkPolicies. Currently
-Geneve, VXLAN, and STT tunnels are supported.
+Antrea can manage the container networking needs of Windows worker nodes.
+
+On Windows, Antrea sets up an overlay network to forward packets between Nodes, while also implementing NetworkPolicies.
+
+Currently Geneve, VXLAN, and STT tunnels are supported.
 
 This page shows how to install antrea-agent on Windows Nodes and register the
 Node to an existing Kubernetes cluster.
 
-For the detailed design of how antrea-agent works on Windows, please refer to
+For a detailed design of how antrea-agent works on Windows, please refer to
 the [design doc](design/windows-design.md).
 
 ### Components that run on Windows
 
-The following components should be configured and run on the Windows Node.
-* [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
+The following components need to be configured to run antrea as the networking provider for a Windows Node:
+
+* standard [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
```

Comment:
Is "standard"  a generic word used to describe Kubernetes components? What about "[standard Kubnernetes components]"?

Reply:
fixed